### DISCUSSION TITLE
Filter by TestCase on the dashboard

### DISCUSSION THREAD

**Discussion by Tonicku** [reportportal/reportportal#1640] [Ideas]
are you planning to add the filter functionality on the Dashbaord ?
goal:  in addition to have an overview of all tests, to observe results on specific TestCase.
avoid to create/(copy/paste) filters and create new dashboard

**Comment by DzmitryHumianiuk** [comment]
@Tonicku yes, we plan to add possibilities to filter out at the test cases level for the dashboards.
This will come with a newly reworked UI.
